index.js

```diff
@@ -59,7 +65,13 @@ module.exports = (input, opts) => {
 				task: () => del('node_modules')
 			},
 			{
-				title: 'Installing dependencies',
+				title: 'Installing dependencies using Yarn',
+				enabled: () => opts.yarn === true,
+				task: () => exec('yarn', ['install'])
```

Comment:
> doing a yarn install can generate a different lockfile

I don't think it is, that's the whole idea of the lock file... Unless you run `yarn upgrade`. Or am I wrong? I never experienced a changed yarn.lock file after running `yarn`.